Peer.Connectors/AzdoPullRequestSource.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.TeamFoundation.Core.WebApi;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
+using Peer.Domain;
+using Peer.Domain.Models;
+using Peer.User;
+
+namespace Peer.Connectors
+{
+    public class AzdoPullRequestSource : IPullRequestSource<AzdoPullRequestSource>
+    {
+        private readonly AppConfig _config;
+        private static IClient<UserAzdoClient> _userClient;
+
+        public AzdoPullRequestSource(AppConfig config, IClient<UserAzdoClient> userClient)
+        {
+            _config = config;
+            _userClient = userClient;
+        }
+
+        public async Task<IEnumerable<PeerPullRequest>> FetchPullRequestsAsync()
+        {
+
+            var userclient = _userClient.CreateClient();
+            var azClient = userclient.AzClient;
+            var azProjClient = userclient.AzProjClient;
+
+            var projects = await azProjClient.GetProjects();
+
+            var prs = new List<GitPullRequest>();
+
+            foreach (TeamProjectReference project in projects)
```

Comment:
Does `var` not work here? I remember the DTE collections having issues with the types of iterable members, maybe the same thing is happening here?